homeassistant/components/climate/ephember.py

```diff
@@ -93,12 +98,40 @@ def target_temperature_step(self):
 
         return 1
 
+    @property
+    def device_state_attributes(self):
+        """Show Device Attributes."""
+        attributes = {
+            'currently_active': self._zone['isCurrentlyActive']
+        }
+        return attributes
+
     @property
     def current_operation(self):
         """Return current operation ie. heat, cool, idle."""
+        mode = self._ember.get_zone_mode(self._zone_name)
+        return self.map_mode_eph_hass(mode)
+
+    @property
+    def operation_list(self):
+        """Return the supported operations."""
+        return [STATE_AUTO, STATE_ALL_DAY, STATE_HEAT, STATE_OFF]
```

Comment:
Move this to a module level constant and return that here.

Reply:
No operation modes that are not defined in the climate base component are allowed. Please remove `STATE_ALL_DAY`. New modes must be proposed and accepted via an issue in our architecture repo.

Were currently reviewing the climate component in our architecture repo.